include/znc/Modules.h

```diff
@@ -356,7 +359,7 @@ class CModCommand {
 class CModule {
 public:
 	CModule(ModHandle pDLL, CUser* pUser, CIRCNetwork* pNetwork, const CString& sModName,
-			const CString& sDataDir);
+			const CString& sDataDir, CModInfo::EModuleType eType = CModInfo::NetworkModule);
```

Comment:
I'm still dreaming about the day we start seriously breaking things and putting the API in shape :P